### PR TITLE
Project 2 from Jean-François.

### PR DESCRIPTION
**Pull request by jfsubrini**
Good reading and please make comments.
I'm sure there are at least two things I can improve. I let you figure it out !
Good vacation. See you in September.

### CODE REVIEW

**Comment by timoguic** [comment]
When your description text spans over more than one line, you have to go to the next line and indent the text with 4 spaces, as you can see in the comments and other students' files. Please adjust and commit again :-)

File: content/students/jeanfrancois.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+# One line.
+# If you need more space, go to the next line and add 4 spaces on the left, as in 'description'.
+objective: Prendre encore et toujours une nouveau chemin vers les joyeuses pistes de la connaissance.
+short_description: D’Andalousie, je prends cette invitation au voyage au pays du code comme un nouveau défi. Yep !
+
+# don't touch that
+template: students
+description:
+    Bourlingueur de grand chemin, vivant actuellement sous le soleil et la douceur de l’Andalousie. J’ai découvert récemment un goût pour le code qui me mènera peut-être vers de nouveaux horizons professionnels. Mon attrait pour la nature et les animaux sauvages m’a fait choisir Python. Non, je blague ! Mon parcours, entre la Finance, l’entreprenariat et les voyages, paraît aussi sinueux que le déplacement de notre reptile, mais au fond, je tâche de garder le cap, tout droit.
```
Comment (timoguic) [inline content/students/jeanfrancois.md]: This part needs adjustments.

Comment (jfsubrini) [inline content/students/jeanfrancois.md]: I don't understand why it's like this ?!? In my .md field it's all right !
capture d ecran 2017-08-24 a 16 34 59

Comment (jfsubrini) [inline content/students/jeanfrancois.md]: Moreover, you can see it in the screenshot projet_2.png

**Pull request merged**